rating.rb

```diff
@@ -0,0 +1,25 @@
+require_relative 'file_handler'
+require_relative 'orders'
+
+class Rating
+  attr_reader :orders
+
+  def initialize(orders = FileHandler.new('orders').parse_file)
+    @orders = orders
+  end
+
+  def top_ids(ntimes, orders_key)
```

Comment:
ntimes -> n_times I ruby we are using snake case

Reply:
Fixed